ui/app/models/aws/root-config.js

```diff
@@ -45,23 +45,33 @@ export default class AwsRootConfig extends Model {
   iamEndpoint;
   @attr('string', { label: 'STS endpoint' }) stsEndpoint;
   @attr('number', {
-    label: 'Maximum retries',
     subText: 'Number of max retries the client should use for recoverable errors. Default is -1.',
   })
   maxRetries;
 
```

Comment:
nothing changed here, just restructuring so the pattern matches the pattern in azure and gcp.